allennlp/training/metrics/metric.py

```diff
@@ -11,7 +12,10 @@ class Metric(Registrable):
     """
 
     def __call__(
-        self, predictions: torch.Tensor, gold_labels: torch.Tensor, mask: Optional[torch.Tensor]
+        self,
+        predictions: torch.Tensor,
+        gold_labels: torch.Tensor,
+        mask: Optional[torch.Tensor] = None,
```

Comment:
I changed this because mypy was complaining about the line that says `metric(logits, label)`, stating there's a missing param, which is actually true. I could have set it to `None`, but then saw the documentation and it says "default = None", which is actually false, so I changed it.